app/assets/stylesheets/peoplefinder/peoplefinder-ie7.scss

```diff
@@ -0,0 +1,116 @@
+@import '_colours';
+// IE7 styles
+$blackish: #6F777B;
+$darkblue: #005EA5;
+$turqoise: #93cfca  ;
+$opaqueblue: rgba(#2B8CC4, 0.25);
+$paleblue: rgba(#9BB7CE, 0.5);
+$error-text: #DF3034;
+$error-border: $error-text;
+$error-bg: #FBE1E1;
+$palegreen: #c9d8bd;
+$notice-bg: $palegreen;
+$pink-bg: #FEF7F7;
+$grey-bg: #DEE0E2;
+$green: #006435;
+$border-color: #bfc1c3;
+
+// TODO: sort out 3 misaligned fields on /people/mark-ford/edit?prompt=profile
+
+// layout stuff
+.header-app-name a{
+  color: white;
+  text-decoration: none;
+}
+.grid-wrapper{
+  clear:both;
+}
+h1, h2 {
+  margin: 30px 0;
+  border-bottom: 1px solid #6f777b;
+}
+
+
+.editing-alert {
+  background-color: $turqoise;
+  text-align: center;
+  padding:1em;
+  height: auto;
+  display: block;
+}
```

Comment:
actually the problem was IE8 not understanding RGBA. I'm overriding the variable now